File: EEDU-Backend/src/main/java/de/gaz/eedu/user/illnessnotifications/model/ReducedIllnessNotificationModel.java
package de.gaz.eedu.user.illnessnotifications.model;

import de.gaz.eedu.user.illnessnotifications.IllnessNotificationStatus;
import org.jetbrains.annotations.NotNull;

public record ReducedIllnessNotificationModel(@NotNull Long id, @NotNull Long userId, @NotNull IllnessNotificationStatus status,
                                              @NotNull Long timestamp, @NotNull Long expirationTime)
{
}

File: EEDU-Frontend/src/app/illness-notification/model/reduced-illness-notification-model.ts
import {IllnessNotificationStatus} from "../illness-notification-status";

export interface ReducedIllnessNotificationModel {
    readonly id: bigint;
    readonly userId: bigint;
    readonly status: IllnessNotificationStatus;
    readonly timestamp: bigint;
    readonly expirationTime: bigint;
}
